sormas-backend/src/main/java/de/symeda/sormas/backend/customizableenum/CustomizableEnumFacadeEjb.java

```diff
@@ -91,6 +91,10 @@ public class CustomizableEnumFacadeEjb
 	 * enum values as its values.
 	 */
 	private final EnumMap<CustomizableEnumType, Map<String, Map<String, Object>>> enumProperties = new EnumMap<>(CustomizableEnumType.class);
+	/**
+	 * Maps a customizable enum type to a map with all enum values of this type as its keys and their active status as its values.
+	 */
+	private final EnumMap<CustomizableEnumType, Map<String, Boolean>> enumActiveStatuses = new EnumMap<>(CustomizableEnumType.class);
```

Comment:
Would be cleaner to change the above field to include the active flag too, I mean use a custom class instead of `Map<String, Map<String, Object>>`

Or event better refactor the `enumValues` field

Keeping this structure the next developer will introduce yet another map for another field on the customizable enum, and in my opinion it just makes the code more complicated and harder to read and understand